### PR TITLE
Updated badge component

### PR DESCRIPTION
**Pull request by sohan9819**
Updated the badge component so that  now it is more easy to edit the data

### CODE REVIEW

File: css/components/badges.css [hunk 1]
```diff
@@ -11,10 +11,10 @@
...
   position: relative;
 }
-.badge__basic-content-s,
-.badge__basic-content-m,
-.badge__basic-content-l,
-.badge__basic-content-xl {
+.badge__basic-content-wrapper,
+.badge__basic-content-wrapper,
+.badge__basic-content-wrapper,
+.badge__basic-content-wrapper {
```
Comment (LordUttam) [inline css/components/badges.css:17]: If all four are same why do you need four? Must have been a mistake made out of hurry

File: css/components/badges.css [hunk 2]
```diff
@@ -181,8 +178,7 @@
 }
```
Comment (LordUttam) [inline css/components/badges.css:178]: If you are only changing scale with transform for all badge sizes, why not make code shorter by bringing all common styles in a seperate class?

**Review by LordUttam — APPROVED** [review]
A minor mistake and a suggestion. Rest looks good to me.

**Review by TusharJade — APPROVED** [review]
UI is awesome. Just change what LordUttam is showing. Other than that everything looks great. Good to go for merge.

File: css/components/badges.css (hunk 1 above)
Comment (sohan9819) [reply css/components/badges.css:17]: Yeah , thanks for pointing out will fix it soon 👍

**Comment by sohan9819** [comment]
Done with the suggested changes

**Pull request merged**